src/TodoListItem.js

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 
-const TodoListItem = (props) => (
-  <li>
-    <span>{props.todo}</span>
-  </li>
-);
+const TodoListItem = ({ listID, todo }) => {
+  // console.log(todo);
+  if (!todo) {
```

Comment:
once you initialize your toDoList as an empty array, you will not have to check if it is null